### ISSUE TITLE
25-test_req.t broke in master on NonStop with explicit start/end dates

### ISSUE THREAD

**Issue by rsbeckerca** [openssl/openssl#24188]
This looks like a new regression post-3.3. Looking for help resolving it.
```
25-test_req.t ........................... 101/?
You are about to be asked to enter information that will be incorporated
into your certificate request.
What you are about to enter is what is called a Distinguished Name or a DN.
There are quite a few fields but you can leave some blank
For some fields there will be a default value,
If you enter '.', the field will be left blank.
-----
C field []:AU
SP field []:
L field []:Brisbane
O field []:CryptSoft Pty Ltd
OU field []:.
CN field []:Eric Young
email field []:eay@mincom.oz.au
../../util/wrap.pl ../../apps/openssl req -x509 -new -text -config ../../test/test.cnf -key ../../test/testrsa.pem -not_before today -not_after today -out self-signed_explicit_date.pem => 0
not ok 109 - explicit start and end dates
# ------------------------------------------------------------------------------
#   Failed test 'explicit start and end dates'
#   at test/recipes/25-test_req.t line 614.
25-test_req.t ........................... Dubious, test returned 1 (wstat 256, 0x100)
```

**Comment by nhorman** [comment]
can you bisect to see when this was introduced?

**Comment by t8m** [comment]
I've just merged a fix to the explicit start/end date tests by Viktor. Can you please try fresh master branch?

**Comment by rsbeckerca** [comment]
Fixed at 4e3c1e6206
